src/EditorFeatures/CSharpTest/Diagnostics/UseImplicitOrExplicitType/UseImplicitTypeTests.cs

```diff
@@ -1244,35 +1233,33 @@ public void Process()
         }
 
         [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType)]
-        public async Task SuggestVarWhereTypeIsEvident_IsExpression()
+        public async Task SuggestVar_BuiltInTypesRulePrecedesOverTypeIsApparentRule2()
         {
-            await TestInRegularAndScriptAsync(
+            await TestMissingInRegularAndScriptAsync(
 @"using System;
 
 class C
 {
     public void Process()
     {
-        A a = new A();
-        [|Boolean|] s = a is IInterface;
+        object o = int.MaxValue;
+        [|Int32|] i = (Int32)o;
     }
-}
-
-class A : IInterface
-{
-}
+}", new TestParameters(options: ImplicitTypeWhereApparent()));
+        }
 
-interface IInterface
-{
-}",
+        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsUseImplicitType)]
+        public async Task SuggestVarWhereTypeIsEvident_IsExpression()
```

Comment:
❗️ This test is no longer doing what it was before. The body of process could be this to preserve intent:

```
A a = new A();
IInterface s = a as IInterface;
```